lib/shifts.rb

```diff
@@ -2,65 +2,49 @@
 
 class Shifts
 
-    def initialize
-      @shifts = shifts
-      @keys = keys
-      @offsets = offsets
-    end
-
     def random_number
       rand(99999)
     end
 
-    def padded_random_number
-      sprintf("%05d", random_number)
+    def padded_random_number(number = random_number)
```

Comment:
Base implementation of an optional argument for all methods that concern a random number